app/controllers/games_controller.rb

```diff
@@ -1,3 +1,17 @@
+
+module GamesHelper
+  
+  def cell_black(row, column)
+    if row % 2 == 0 && column % 2 == 0 || row % 2 != 0 && column % 2 != 0
+      return true
+    else
+      return false
+    end
+  end
+
+end
+
+
```

Comment:
I was reading up on having a helper method in the controller and it seems like the way most people recommend is to declare a helper_method under private in the GamesController class. So maybe something like:

`private
helper_method :cell_black
def cell_black(row, column)
end`

But maybe @ilyakrasnov can suggest the proper way. This is what I was reading (see under "Old Answer"): https://stackoverflow.com/questions/10714119/ruby-on-rails-call-controller-method-from-a-view